### DISCUSSION TITLE
⭐️Starstruck Exchange⭐️ Repository to generate faces that do not exist

### DISCUSSION THREAD

**Discussion by bytesleo** [Schweinepriester/github-profile-achievements#790] [Q&A]
Hello Guys, I show you this repo to generate people because they don't exist in javascript

https://github.com/kevoj/thispersondoesnotexist-js

**Comment by eglhub** [comment]
wonderful

**Comment by santoshpanda1995** [comment]
Great repo !! The use of GANs are really changing the world.

**Comment by ArjavJain-27** [comment]
This is a really interesting project! I like how it wraps the `thispersondoesnotexist.com` API into a simple JavaScript package.

The get Image() method makes it pretty easy to integrate generated avatars into other applications, and I also like that it supports both file and Base64 output.

This could be useful for testing profile/avatar systems without using real people's photos. Definitely going to explore the repo and see how the API wrapper is implemented.
